apm-agent-core/src/test/java/co/elastic/apm/agent/logging/LoggingConfigurationTest.java

```diff
@@ -237,7 +237,7 @@ public LoggerContext getContext(String fqcn, ClassLoader loader, boolean current
                 }
                 LoggerContext loggerContext = contextMap.get(loader);
                 if (loggerContext == null) {
-                    org.apache.logging.log4j.core.LoggerContext loggerContextImpl = new org.apache.logging.log4j.core.LoggerContext(Objects.toString(loader));
+                    org.apache.logging.log4j.core.LoggerContext loggerContextImpl = new org.apache.logging.log4j.core.LoggerContext(loader.getClass().getName());
```

Comment:
```suggestion
                    org.apache.logging.log4j.core.LoggerContext loggerContextImpl = new org.apache.logging.log4j.core.LoggerContext(loader.getClass().getName() + String.valueOf(System.identityHashCode(loader)));
```
We want a context per CL instance, not per CL type